clean_air/data/filter_data_demo.ipynb

```diff
@@ -0,0 +1,367 @@
+       " '/net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/metaone.yaml',\n",
+       " '/net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/station_metadata.yaml'}"
+      ]
+     },
+     "execution_count": 113,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "m.get_filtered_data_subsets()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 114,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "  \u001b[1;30m    Obs Level:GROUND  Turn off all datasets containing Ground\n",
+      "\u001b[0;31m Setting the filter to off for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/metaone.yaml, value: False\n",
+      "\u001b[0m\n",
+      "\u001b[0;31m Setting the filter to off for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/station_metadata.yaml, value: False\n",
+      "\u001b[0m\n",
+      "\u001b[0;31m Setting the filter to off for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/antartica_metadata.yaml, value: False\n",
+      "\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('\\n  \\033[1;30m    Obs Level:GROUND', ' Turn off all datasets containing Ground')\n",
+    "# Turn a filter off using obs level\n",
+    "m.set_observation_level_filter(False, 'ground')\n",
+    "\n",
+    "#m.is_filter_on('/net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/station_metadata.yaml')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 115,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[1;30m     Obs Level:GROUND  Turn on all datasets containing Ground\n",
+      "\u001b[0;32m Setting the filter to on for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/metaone.yaml, value: True\n",
+      "\u001b[0m\n",
+      "\u001b[0;32m Setting the filter to on for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/station_metadata.yaml, value: True\n",
+      "\u001b[0m\n",
+      "\u001b[0;32m Setting the filter to on for key: /net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/antartica_metadata.yaml, value: True\n",
+      "\u001b[0m\n",
+      "\u001b[0;32m Filter ON, value = True  \n",
+      "\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "True"
+      ]
+     },
+     "execution_count": 115,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Turn a filter on using obs level\n",
+    "print('\\033[1;30m     Obs Level:GROUND', ' Turn on all datasets containing Ground')\n",
+    "m.set_observation_level_filter(True, 'ground')\n",
+    "\n",
+    "m.is_filter_on('/net/home/h05/clucas/PycharmProjects/CleanAirProject/clean_air/data/station_metadata.yaml')"
```

Comment:
I think there's a few occurrences of `is_filter_on` that need to be changed to `is_match` in this notebook

Reply:
Removed this file as no longer using.